Yoast/YoastSEO.js
# Add French and Spanish Flesch Reading Ease Assessment

## Summary

This PR can be summarized in the following changelog entry:

* Adds Flesch reading ease assessment for French.
* Adds Flesch reading ease assessment for Spanish.

## Test instructions

The code in this PR has also already been reviewed and acceptance-tested, so a quick acceptance test should be sufficient to make sure everyhing works as expected.

This PR can be tested by following these steps:

* Make a beta version using the beta script. For Yoast SEO, use `trunk`, for YoastSEO.js, use `feature/languages`.

### French
* In your testing environment, set the language to Spanish.
* Write a Spanish text.
* Confirm that the Flesch Reading score appears.

### Spanish
* In your testing environment, set the language to Spanish.
* Write a Spanish text.
* Confirm that the Flesch Reading score appears.
